xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -143,6 +171,37 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     this.config = unmarshaller.unmarshal(dom, configTypeClass).getValue();
   }
 
+  private void substituteSystemProperties(final Element dom) {
```

Comment:
This could be static, and also wouldn't it be cleaner if this method returned the resultant dom?